src/objects.js

```diff
@@ -1,42 +1,53 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name
+    age: age
```

Comment:
This is missing a comma so is not running:
```
  return {
    name: name,
    age: age
  };
```
There is a new shorthand way of writing this too, when the keys and values are the same you can just write them singly like this:
```
  return {
    name,
    age
  };
```